src/main/java/com/greenfoxacademy/opal/kalendaryo/kalendaryo/controllers/NotificationController.java

```diff
@@ -46,7 +46,7 @@ public String getAccessToken(@RequestParam String authCode) throws IOException {
     public UserModel getRegistration(@RequestBody AuthModel authModel) throws IOException {
```

Comment:
@dzlica let's start using naming convention for methods: `httpmethodEndpointname`